.scripts/ci_configure_build_test.py

```diff
@@ -39,31 +64,92 @@ def main():
         generator = 'Unix Makefiles'
         buildflags = 'VERBOSE=1'
         topdir = os.getcwd()
+    return generator, buildflags, topdir, is_visual_studio
+
+
+def handle_errors(errors):
+    if errors is None:
+        sys.stdout.write('OK\n')
+        return_code = 0
+    else:
+        sys.stdout.write('FAILED\n')
+        sys.stderr.write(errors + '\n')
+        return_code = 1
+    sys.stdout.flush()
+    sys.stderr.flush()
+    return return_code
 
+
+def parse_yaml():
+    '''
+    If recipe directory contains a file called config.yml, we parse it.
+    '''
+    import yaml
+    import sys
+    file_name = 'config.yml'
+    if os.path.isfile(file_name):
+        with open(file_name, 'r') as f:
+            try:
+                config = yaml.load(f, yaml.SafeLoader)
+            except yaml.YAMLError as exc:
+                print(exc)
+                sys.exit(-1)
+        return config
+    return {}
+
+
+def main():
+    recipes = get_list_of_recipes_to_run()
+    generator, buildflags, topdir, is_visual_studio = get_env_variables()
+
+    return_code = 0
     for recipe in recipes:
         recipe_dir = os.path.abspath(recipe)
         os.chdir(recipe_dir)
         # Glob examples
         examples = [e for e in sorted(glob.glob('*-example'))]
+
+        # TODO we need to get rid of this
         # Remove Fortran examples if generator is Visual Studio
         if is_visual_studio:
             examples = filter(lambda x: 'fortran' not in x, examples)
+
         for example in examples:
+
             os.chdir(os.path.abspath(example))
-            print('{} and {}'.format(recipe, example))
-            # Configure
-            configure = run_command(['cmake', '-H.',
-                                     '-Bbuild', '-G' + generator])
-            [print(x, end="") for x in configure]
+            sys.stdout.write('{}/{}\n'.format(recipe, example))
```

Comment:
Can we parse also the `README.md` one directory up to get the name of the recipe and print it out?

Reply:
Good idea, will append this to this PR.